tests/broker-client.test.ts

```
@@ -0,0 +1,70 @@
+import { beforeAll, describe, expect, it } from 'vitest';
+import { brokerClient } from '../src/broker-client';
+import { HibitNetwork, ChainId, ChainAssetType, ChainNetwork, Chain } from '../src';
```

ESLint

[error] 'ChainId' is defined but never used.

(@typescript-eslint/no-unused-vars)